packaging/debian/rules

```diff
@@ -15,6 +15,7 @@ override_dh_auto_install:
 	# Binary-only package.
 	dh_auto_install -- --no-source
 	mv debian/google-osconfig-agent/usr/bin/osconfig debian/google-osconfig-agent/usr/bin/google_osconfig_agent
+	dh_installdirs /etc/osconfig
```

Comment:
if we're declaring that this directory should always exist when osconfig agent is installed, we should just ship a file into it and not have type-specific logic for creating empty directories. it can be a readme file.

Reply:
removed